src/parser/monk/mistweaver/modules/features/StatValues.tsx

```diff
@@ -7,21 +7,25 @@ import StatTracker from 'parser/shared/modules/StatTracker';
 import CritEffectBonus from 'parser/shared/modules/helpers/CritEffectBonus';
 
 import SPELL_INFO from './StatValuesSpellInfo';
+import { HealEvent } from 'parser/core/Events';
+import HealingValue from 'parser/shared/modules/HealingValue';
 
 /**
  * Mistweaver Monk Stat Values
-**/
-
+ */
 class StatValues extends BaseHealerStatValues {
   static dependencies = {
     statTracker: StatTracker,
     critEffectBonus: CritEffectBonus,
   };
 
-  spellInfo = SPELL_INFO;
-  qeLive = true;
+  protected statTracker!: StatTracker;
+  protected critEffectBonus!: CritEffectBonus;
+
+  spellInfo: {} = SPELL_INFO;
```

Comment:
`spellInfo: unknown`

Reply:
Could you expand on this as I'm not following what you mean exactly

Reply:
When you don't know (or dont want to figure out) the shape of something, and you aren't doing much with it (i dont see it referenced anywhere in here), you can use the `unknown` type. It's like a typesafe `any` -- you can assign anything to it, but trying to read any values of it give you a compile warning